pkg/mount/mountinfo.go

```diff
@@ -1,40 +1,22 @@
 package mount // import "github.com/docker/docker/pkg/mount"
 
-// Info reveals information about a particular mounted filesystem. This
-// struct is populated from the content in the /proc/<pid>/mountinfo file.
-type Info struct {
-	// ID is a unique identifier of the mount (may be reused after umount).
-	ID int
-
-	// Parent indicates the ID of the mount parent (or of self for the top of the
-	// mount tree).
-	Parent int
-
-	// Major indicates one half of the device ID which identifies the device class.
-	Major int
-
-	// Minor indicates one half of the device ID which identifies a specific
-	// instance of device.
-	Minor int
-
-	// Root of the mount within the filesystem.
-	Root string
-
-	// Mountpoint indicates the mount point relative to the process's root.
-	Mountpoint string
-
-	// Opts represents mount-specific options.
-	Opts string
-
-	// Optional represents optional fields.
-	Optional string
-
-	// Fstype indicates the type of filesystem, such as EXT3.
-	Fstype string
-
-	// Source indicates filesystem specific information or "none".
-	Source string
-
-	// VfsOpts represents per super block options.
-	VfsOpts string
-}
+import (
+	"github.com/moby/sys/mountinfo"
+)
+
+//nolint:golint
+type (
+	FilterFunc = mountinfo.FilterFunc
```

Comment:
same comment here (add "deprecated" comments)

Reply:
Explained it in the other PR -- I mark the whole package as deprecated only after we really switch to using moby/sys, otherwise linter would complain. This is also why I split this into two PRs (to validate I don't break it in the middle of the changes, also to validate the to-be-deprecated package can still be used).

Reply:
this is made deprecated in part II (https://github.com/moby/moby/pull/40694)